cmd/kubeadm/app/phases/controlplane/manifests_test.go

```diff
@@ -138,6 +141,56 @@ func TestCreateStaticPodFilesAndWrappers(t *testing.T) {
 	}
 }
 
+func TestCreateStaticPodFilesKustomize(t *testing.T) {
+	// Create temp folder for the test case
+	tmpdir := testutil.SetupTempDir(t)
```

Comment:
non-blocking: we should stop importing `k8s.io/kubernetes/cmd/kubeadm/test"` in `.../app`.